src/main/java/io/buildpacks/example/sample/GreetingController.java

```diff
@@ -7,8 +7,14 @@
 @RestController
```

Comment:
1. Error handling: 
   - The `throwError` method in the `GreetingController` class throws a `RuntimeException` with a message indicating it's a deliberate error for PR review. While this can be useful for testing error handling, it's not recommended to keep such deliberate errors in production code.